### PR TITLE
Syndicate Decoy Bundle + Snap Pops

### PR DESCRIPTION
**Pull request by musicmanvr**
## About the PR

This PR adds a few new toys for Syndicate operatives, as well as a new addition to the toy box, snap pops!
## Why / Balance

Most Syndicate equipment is centered around the more direct side of combat, whereas the additions that I want to add help nuclear operatives to potentially outwit their foes with decoys and trickery.

This includes:
- Five Syndicate decoy balloons, one for each type of Syndicate. (Operative, Agent, Elite, Juggernaut and Commander)
- Two new types of "dummy" bomb, which look exactly like a grenade and minibomb but flash and fart when they explode.
- Snap Pops, little bags filled with flammable stuff that makes an extremely low damage explosion when they get thrown.
- Syndicate Commander trench whistle, a way to rally the troops!
## Technical details

- Syndicate decoy balloons can be beaten to "crit" and pop, or can be popped by security's anti-inflatable needle.
- In addition, these decoy balloons each have a light identical in function to their hardsuit counterparts minus the juggernaut suit which gets a standard light. Inspecting the balloons will reveal their tricks, and they can be packed up into boxes by folding them.

- Dummy bombs make a non-blinding flash and fart noise when they explode, but sound the same during their countdown. Inspecting them will reveal their trickery.

- Snap Pops do 100x the minimum damage of a normal explosion, which is roughly 0.15 damage per pop before resistances.
- Snap Pops can be used to destroy light tubes.

- Syndicate Commanders now spawn with a trench whistle, Avanti!
## Media

image

- [ X ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

🆑 
- add: Added the Syndicate decoy bundle, a creative way to trick your foes.
- add: Snap pops have been added to the toy crate.
- add: Syndicate Commanders now have a trench whistle, Avanti!

### CODE REVIEW

**Comment by K-Dynamic** [comment]
Leftmost nukie operative balloon appears to have commander helmet as opposed to regular helmet

**Comment by Hmeister-real** [comment]
this looks so fun, best pr of 2024 so far imo

**Comment by mirrorcult** [comment]
is there an intended difference between the trickybomb / decoy grenade

**Comment by musicmanvr** [comment]
> is there an intended difference between the trickybomb / decoy grenade

the trick grenades look exactly like the real thing but make a harmless flash and fart noise instead of an explosion
they're named differently to prevent people from attempting to throw real grenades that actually explode

**Comment by mirrorcult** [comment]
i mean between the trickybomb & the decoy grenade, they seem like pretty much the same thing but with different sprites

**Comment by musicmanvr** [comment]
> i mean between the trickybomb & the decoy grenade, they seem like pretty much the same thing but with different sprites

there isn't any difference other than mimicing their associated type of grenade, I didn't really think they needed to be anything too too special other than a distraction tool

File: Resources/Locale/en-US/store/uplink-catalog.ftl [hunk 1]
```diff
@@ -189,6 +189,9 @@ uplink-observation-kit-desc = Includes surveillance camera monitor board and sec
 uplink-emp-kit-name = Electrical Disruptor Kit
 uplink-emp-kit-desc = The ultimate reversal on energy-based weaponry: Disables disablers, stuns stunbatons, discharges laser guns! Contains 3 EMP grenades and an EMP implanter. Note: Does not disrupt actual firearms.
 
+uplink-decoy-kit-name = Decoy Kit
+uplink-decoy-kit-desc = State of the art distraction technology straight from RND. Comes with five realistic balloons, four decoy grenades, and some snap poppers!
```
Comment (FairlySadPanda) [inline Resources/Locale/en-US/store/uplink-catalog.ftl:193]: `uplink-decoy-kit-desc = State-of-the-art distraction technology straight from RND. Comes with five realistic balloons, four decoy grenades, and some snap poppers!`

**Review by FairlySadPanda — COMMENTED** [review]
Small grammar fix (sorry!)

**Comment by Kadeo64** [comment]
now crew needs decoy stuff, would be funny if we could put the nuke disk on a captain decoy and stuff 4 seismic charges inside of it

**Comment by musicmanvr** [comment]
Just to clarify, snap pops make a cosmetic explosion that do not set off other explosives.

**Comment by DangerRevolution** [comment]
Very cool o7

File: Resources/Textures/Clothing/Neck/Misc/whistles.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Drawn by Mike.",
```
[the remark this answers is not in the document]
Comment (notquitehadouken) [reply Resources/Textures/Clothing/Neck/Misc/whistles.rsi/meta.json]: M i k e

[the remark this answers is not in the document]
Comment (musicmanvr) [reply Resources/Textures/Clothing/Neck/Misc/whistles.rsi/meta.json]: this has been fixed

**Pull request merged**